via/static/scripts/video_player/index.tsx

```diff
@@ -16,28 +13,18 @@ export function init() {
   }
 
   const {
+    api,
     client_config: clientConfig,
     client_src: clientSrc,
-
-    // Ignored until backend is able to provide real transcript data.
-    // transcript,
-
     video_id: videoId,
   } = readConfig();
 
-  // Pre-fetched transcript for testing. From the video
-  // https://www.youtube.com/watch?v=x8TO-nrUtSI.
-  const transcript: TranscriptData = { segments: sampleTranscript };
-
-  // Group segments together for better readability.
-  transcript.segments = mergeSegments(transcript.segments, 3);
-
   render(
     <VideoPlayerApp
       videoId={videoId}
       clientConfig={clientConfig}
       clientSrc={clientSrc}
-      transcript={transcript}
+      transcriptSource={api.transcript}
```

Comment:
Let me ensure I've got this right: `VideoPlayerApp`'s `transcriptSource` prop is typed `APIMethod | TranscriptData`, but the app's entry point will only ever deal in `APIMethod`. Is the `TranscriptData` type allowed entirely as a test affordance or do you anticipate it being more generally useful?

_Nevermind_, the PR description answers this question:

> In the actual app an API call is always used, but the pre-fetched transcript is useful in tests.

Given my distraction here, it might be useful to add a comment somewhere to this effect. Maybe?